cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,95 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if billingAccount == "" && !interactive.Enabled() {
+		billingAccount = aws.GetJumpAccount("local")
```

Comment:
Minor comment. In `cmd/create/ocmrole/cmd.go` the env is from below and passed in to the GetJumpAccount
```
	env, err := ocm.GetEnv()
```
@cristianoveiga is it necessary to do this or just leave it hardcode as local?

Reply:
nice catch @mikeshng . We need to use the ` awsCreator.AccountID` instead of the hard coded one.

See https://github.com/openshift/rosa/pull/1227/files#diff-943c0ab57b82c7028652df55e1d49e4f4e4dc41fa094dc4a9b3f2c35887d44b5R712-R715

Reply:
Refactored.